rails-footnotes.gemspec

```diff
@@ -6,23 +6,16 @@ Gem::Specification.new do |s|
   s.name        = "rails-footnotes"
   s.version     = Footnotes::VERSION
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Roman V. Babenko", "José Valim", "Keenan Brock", "Duane Johnson", "Adrien Siami"]
-  s.email       = ["romanvbabenko@gmail.com"]
-  s.homepage    = "http://github.com/josevalim/rails-footnotes"
+  s.authors     = ["Roman V. Babenko", "José Valim", "Keenan Brock", "Duane Johnson", "Adrien Siami", "André Arko"]
+  s.email       = ["andre@arko.net"]
+  s.homepage    = "http://github.com/indirect/rails-footnotes"
```

Comment:
I can work out the transfer of the repository under your username, would you want it?

Reply:
Sure, that works for me. Maybe we merge this PR, and then transfer it after that?